### PR TITLE
feat(miranum-modeler): miranum modeler docs

### PR DESCRIPTION
**Pull request by peterhnm**
## Description 
Add a documentation for the Miranum Modeler

## Checklist 
- [x] No obvious errors or bugs (`npm run start` works)
- [x] Followed the [Style Guides](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/technical-writing-styleguide.md)
- [x] Using a semantic commit messages as described in [here](https://gist.github.com/joshbuchea/6f47e86d2510bce28f8e7f42ae84c716)

### CODE REVIEW

File: docs/components/miranum-ide/miranum-modeler.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+The plugin targets the creation of Camunda Platform 7 compatible BPMN diagrams and enhance them with technical attributes.
+It uses the Open Source Projects [bpmn-js](https://bpmn.io/toolkit/bpmn-js/) and [VS Code](https://code.visualstudio.com/) 
+to make the life of an BPM Developer a little easier.
+
+## Features
+
+### Model your BPMN 2.0 diagram
+
+![create-process.gif](@site/docs/components/miranum-ide/static/img/miranum-modeler/create-process.gif)
+
```
Comment (Hafflgav) [inline docs/components/miranum-ide/miranum-modeler.md:22]: 💭 I would prefer to have a list of bullet points explaining what you are showing in the gif right here.

File: docs/components/miranum-ide/miranum-modeler.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+#### Use element templates
+
+![add-element-templates.gif](@site/docs/components/miranum-ide/static/img/miranum-modeler/add-element-templates.gif)
+
```
Comment (Hafflgav) [inline docs/components/miranum-ide/miranum-modeler.md:26]: 💭Same here (Bullet points)

File: docs/components/miranum-ide/miranum-modeler.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+#### Use the integrated text editor for quick fixes
+
+![text-editor.gif](@site/docs/components/miranum-ide/static/img/miranum-modeler/text-editor.gif)
+
```
Comment (Hafflgav) [inline docs/components/miranum-ide/miranum-modeler.md:30]: 💭Same here (Bullet points)

**Review by Hafflgav — CHANGES_REQUESTED** [review]
I would request these changes in favour of accessibility of our documentation. 
But I am up for discussion if you do not think this is a fit. :)

File: docs/components/miranum-ide/miranum-modeler.md (hunk 1 above)
Comment (peterhnm) [reply docs/components/miranum-ide/miranum-modeler.md:25]: I added the description to the gifs.

File: docs/components/miranum-ide/miranum-modeler.md (hunk 3 above)
Comment (peterhnm) [reply docs/components/miranum-ide/miranum-modeler.md:43]: Same here

File: docs/components/miranum-ide/miranum-modeler.md [hunk 4]
```diff
@@ -17,19 +17,47 @@ to make the life of an BPM Developer a little easier.
...
+2. Place a new [element template](https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/about-templates/) inside the new created folder
+3. Assign the element template to a BPMN element by using the menu on the right side
+4. Configure your diagram with the newly added values
 
 ![add-element-templates.gif](@site/docs/components/miranum-ide/static/img/miranum-modeler/add-element-templates.gif)
 
 #### Use the integrated text editor for quick fixes
+1. Click on the button on the navigation bar in the top right corner  
+   ![text-editor-btn](@site/docs/components/miranum-ide/static/img/miranum-modeler/text-editor-btn.png)
+2. This will open the text editor beside the BPMN editor
```
Comment (Hafflgav) [inline docs/components/miranum-ide/miranum-modeler.md:38]: ⛏️  "beside**s**"

File: docs/components/miranum-ide/miranum-modeler.md [hunk 5]
```diff
@@ -17,19 +17,47 @@ to make the life of an BPM Developer a little easier.
...
+2. Start simulating your process.
+   > Note: Not every BPMN element is supported
+3. This will give you a good first impression if your process is executable
+4. Switch back to the editor to modify your process  
+   ![token-sim-off](@site/docs/components/miranum-ide/static/img/miranum-modeler/token-sim-on.png)
+ 
+![text-editor.gif](@site/docs/components/miranum-ide/static/img/miranum-modeler/token-sim.gif)
 
 #### See logging information for errors
+1. Click on the button on the navigation bar in the top right corner
```
Comment (Hafflgav) [inline docs/components/miranum-ide/miranum-modeler.md:58]: ⛏️  "Click ~~on~~ the button on the navigation..."

**Review by Hafflgav — APPROVED** [review]
Looks very good! 💯 
Great work 🚀 

I have seen two nitpicks/typo's which would require some action. 
I approved it anyways though. :)

**Pull request merged**